m3u8-cmdline/m3u8.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
+import os
+import platform
+import json
+import argparse
+
+class config_info:
+  def __init__(self, system):
+    self.directory = os.path.join(os.path.expanduser('~'), '.config','m3u8-cmdline')
+    self.os = system
+    self.path = self.directory + '/config.json' 
+config = config_info(platform.system())
+
+
+def Initialize(args):
+  data = {"streamkey":"",
+          "ffmpeg":"ffmpeg -loglevel warning -reconnect 1 -reconnect_at_eof 1 -reconnect_delay_max 10 -i {0} -codec:a aac -c:v copy -f flv rtmp://ingest.angelthump.com:1935/live/{1}"}
+  
+  if not os.path.isdir(config.directory):
+    try:
+      os.makedirs(config.directory)   
+    except Exception as e:
+      print(e)
```

Comment:
try raising these instead of printing, to allow the caller to handle it (which you already do)